src/dora/kaggle.py

```
@@ -0,0 +1,78 @@
+        """
+        Download a Kaggle dataset from kagglehub and return the path to the downloaded file.
+
+        :param dataset_id: The 'owner/dataset-name' identifier of the dataset to download.
+        :return: The path to the downloaded file.
+        """
+        logging.info(f"Downloading dataset {dataset_id}")
+        try:
+            dataset_path = kagglehub.dataset_download(dataset_id)
+            dataset_download_directory = Path(dataset_path)
+        except RuntimeError as e:
+            raise ValueError(f"Failed to download dataset: {e}") from e
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)